test/TEST-45-REPART/test.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+TEST_DESCRIPTION="test systemd-repart"
+
```

Comment:
One (hopefully) last nickpick from the compatibility side of things: the test should checker whether `systemd-repart` is available before trying to run it (similarly how we treat `TEST-06-SELINUX`, for example). Otherwise, it will fail (as expected):

```
Dec 19 16:09:17 systemd-testsuite testsuite.sh[463]: + systemd-repart /tmp/zzz --empty=yes --dry-run=no --seed=e2a40bf9-73f1-4278-9160-49c031e7aef8
Dec 19 16:09:17 systemd-testsuite testsuite.sh[470]: /testsuite.sh: line 10: systemd-repart: command not found
Dec 19 16:09:17 systemd-testsuite systemd[1]: Received SIGCHLD from PID 463 (testsuite.sh).
Dec 19 16:09:17 systemd-testsuite systemd[1]: Child 463 (testsuite.sh) died (code=exited, status=127/n/a)
Dec 19 16:09:17 systemd-testsuite systemd[1]: testsuite.service: Child 463 belongs to testsuite.service.
Dec 19 16:09:17 systemd-testsuite systemd[1]: testsuite.service: Main process exited, code=exited, status=127/n/a
Dec 19 16:09:17 systemd-testsuite systemd[1]: testsuite.service: Failed with result 'exit-code'.
Dec 19 16:09:17 systemd-testsuite systemd[1]: testsuite.service: Service will not restart (restart setting)
Dec 19 16:09:17 systemd-testsuite systemd[1]: testsuite.service: Changed start -> failed
Dec 19 16:09:17 systemd-testsuite systemd[461]: Operating on architecture: x32
Dec 19 16:09:17 systemd-testsuite systemd[1]: testsuite.service: Job 195 testsuite.service/start finished, result=failed
Dec 19 16:09:17 systemd-testsuite systemd[1]: Failed to start Testsuite service.
```